### PR TITLE
Docs/linux updates

### PR DESCRIPTION
**Pull request by kd8bny**
1. Add some additional configuration info on how to adjust stream resolution using the kwin compositor.
2. Add a mention on how to inhibit the system sleep and idle, while a stream session is running. (Note: this may be a better fit as a feature within MoonDeckStream)

### CODE REVIEW

File: wiki/Buddy-configuration.md [hunk 1]
```diff
@@ -41,6 +41,22 @@ Note: in the Buddy's configuration the screen is referred to as a display due to
 
 Resolution change not supported yet as the [API](https://wayland.app/protocols/wlr-output-management-unstable-v1) is not mature enough and the compositor support is lacking.
 
+*Note:* Although we cannot change the resolution through sunshine and, we can nicely ask the ask the DE compositor to set through the sunshine *Do* and *Undo* actions.
```
Comment (FrogTheFrog) [inline wiki/Buddy-configuration.md]: Should we maybe refer to https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/app_examples.html#changing-resolution-and-refresh-rate instead?

**Comment by FrogTheFrog** [comment]
I will take a look about inhibiting sleep later when I have time. It seems easy enough.
https://github.com/FrogTheFrog/moondeck-buddy/issues/96

**Comment by FrogTheFrog** [comment]
@kd8bny Please try a build from https://github.com/FrogTheFrog/moondeck-buddy/pull/97 whenever you have time. I have added inhibition based on your suggested arguments.

**Comment by kd8bny** [comment]
Awesome! you work fast!
Ill give a shot and report back

**Comment by kd8bny** [comment]
Got it built and ran today!
All looks good and is properly reporting the inhibit

**Comment by FrogTheFrog** [comment]
I will make a new release as soon as I'm done playing with Windows installer (which should be soon'ish).
I've also added a command line option for Buddy to enable/disable autostart since I want the installer to automatically handle it, FYI.

**Comment by kd8bny** [comment]
Ill update this PR, removing the now unneeded inhibit config option

**Comment by FrogTheFrog** [comment]
I have published new release:
https://github.com/FrogTheFrog/moondeck-buddy/releases/tag/v1.6.2

Added one more command line option for better cleanup (`--close-all`) as closing apps in Windows gracefully is hard...
By the time Buddy (called with `--close-all`) exists, all the other instances should already be closed unless linux is keeping the process hostage in a zombie state or something.

File: wiki/Buddy-configuration.md (hunk 1 above)
Comment (kd8bny) [reply wiki/Buddy-configuration.md]: Oh yeah! I was unaware of this documentation. 
I'll update with a link instead

**Comment by kd8bny** [comment]
Final updates here, removing the inhibit and resolution instructions. 
Feel free to close w/o merge if you feel this doesnt really have value anymore.

**Pull request merged**